styles.css

```diff
@@ -472,6 +472,217 @@ a.see-project:focus {
   padding: 1.125rem;
 }
 
+/* Pop-Up-Styles-Section */
+
+.popup-section {
+  display: none;
+  position: fixed;
+  z-index: 1;
+  overflow: auto;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  background-color: #c1c7d0e5;
+  backdrop-filter: blur(0.126rem);
+}
```

Comment:
Hi @CrystallineButterfly 

Your project is great but you have to make some required changes,

- [ ] Please the pop-up is not responsive to mobile view.
### **Take A Look**
![brs](https://user-images.githubusercontent.com/55596970/168127733-d87e8f99-736b-426c-bccb-fddf7e86eb6a.png)

Reply:
HI @CollinsTatang So my code is good and it is responsive, I have another issue, which I am unsure to solve, therefore you don't see my most recent code, I will submit it again after some more attempts to fix it. 

All the best-K

Reply:
OK @CrystallineButterfly 
I will be waiting.

Reply:
Hi @CollinsTatang Are you free to zoom call today? So I can show you visually my issue?

Reply:
Hi @CrystallineButterfly 
You can send me a call link on slack